lib/helpers/pack_hoodie.js

```diff
@@ -105,20 +106,21 @@ module.exports = function (env_config) {
   // Create the `bundleConcat` that we'll use to create the `browserify` bundle.
   fs.writeFileSync(bundleConcat, bundleTmpl + concatPlugins(plugins));
 
-  var b = browserify(bundleConcat);
-
-  return b.bundle({
+  var b = browserify(bundleConcat, {
     standalone: 'Hoodie',
     external: 'jquery'
-  }, function (err, bundle) {
+  });
+
+  b.bundle(function (err, bundle) {
     if (err) { return console.error(err); }
     // Store successful bundle in cache.
     cache = bundle;
     // Clean up! Remove dynamically generated file with concatenated scripts
     // once we are done building the bundle.
-    fs.unlink(bundleConcat, function (err) {
+    fs.unlinkSync(bundleConcat, function (err) {
```

Comment:
> callback:

yeah, that’s a leftover, will be removed in next push

> done callback

[I define it myself](https://github.com/hoodiehq/hoodie-server/pull/363/files#diff-5fb2471ea988b4a1f26dc7fd0f8c0332R72) — No need to handle errors there.